### PR TITLE
Add Jonan

### PR DESCRIPTION
**Pull request by jonanone**
Add jonan to website

### CODE REVIEW

File: partials/team.html [hunk 1]
```diff
@@ -125,6 +125,14 @@ <h4 class="heading">Meet the team</h4>
       })
 
       @@include('./team-member.html', {
+        "name": "Jon Ander Romero Martínez",
+        "id": "jonan",
+        "position": "Developer: Code Knight",
+        "team": "development",
+        "text": "Jonan is a developer with experience working in multiple fields, from backend to frontend to mobile... and beyond! He loves working with all kind of techonolgies and learn new stuff. He is passionate about gaming and never misses a good gaming evening with his mates. His bad jokes will make you laugh, or cry, but probably laugh, right?"
```
Comment (Nahikariii) [inline partials/team.html]: working in / working on 😉

Comment (grahamrhay) [reply partials/team.html]: wat?

Comment (grahamrhay) [reply partials/team.html]: "techonolgies" though

File: partials/team.html [hunk 2]
```diff
@@ -129,7 +129,7 @@ <h4 class="heading">Meet the team</h4>
         "id": "jonan",
         "position": "Developer: Code Knight",
         "team": "development",
-        "text": "Jonan is a developer with experience working on multiple fields, from backend to frontend to mobile... and beyond! He loves working with all kind of techonolgies and learning new stuff. He is passionate about gaming and never misses a good gaming evening with his mates. His bad jokes will make you laugh, or cry, but probably laugh, right?"
+        "text": "Jonan is a developer with experience working on multiple fields, from backend to frontend to mobile... and beyond! He loves working with all kind of technolgies and learning new stuff. He is passionate about gaming and never misses a good gaming evening with his mates. His bad jokes will make you laugh, or cry, but probably laugh, right?"
```
Comment (grahamrhay) [inline partials/team.html]: ha ha, still wrong

Comment (jonanone) [reply partials/team.html]: oh f..k, technolgies xDDDD

**Pull request merged**